src/daemon/instance_settings_handler.cpp

```diff
@@ -153,6 +155,32 @@ void update_disk(const QString& key, const QString& val, mp::VirtualMachine& ins
     }
 }
 
+bool is_bridged(const mp::VMSpecs& spec)
+{
+    for (const auto& network : spec.extra_interfaces)
+    {
+        if (network.auto_mode)
+            return true;
+    }
+
+    return false;
```

Comment:
std algorithm based code can be another alternative, the code is shorter, the readability probably is the same
```
    return std::any_of(spec.extra_interfaces.cbegin(), spec.extra_interfaces.cend(),
                       [](const auto& network) -> bool { return network.auto_mode; });
```

Reply:
Looks nice, thanks! Changed.